controllers/taloscontrolplane_controller.go

```diff
@@ -540,10 +551,7 @@ func (r *TalosControlPlaneReconciler) bootControlPlane(ctx context.Context, clus
 		return ctrl.Result{}, err
 	}
 
-	bootstrapConfig := &tcp.Spec.ControlPlaneConfig.InitConfig
-	if nodeType == controlplaneNode {
-		bootstrapConfig = &tcp.Spec.ControlPlaneConfig.ControlPlaneConfig
-	}
+	bootstrapConfig := &tcp.Spec.ControlPlaneConfig.ControlPlaneConfig
```

Comment:
we should clearly deprecate `initConfig` now, and probably fail if it is there (not empty) while booting the first node?

Reply:
Yeah, that's a good point.